packages/commons/src/main/util/RandomUtil.test.ts

```diff
@@ -29,10 +31,10 @@ describe('RandomUtil', () => {
       const randomLimit = 10;
       for (let index = 0; index < iterations; index++) {
         if (RandomUtil.randomInt(randomLimit) > randomLimit) {
-          fail(`randomInt() exeeded limit "${randomLimit}"`);
```

Comment:
Why using throw here? Fail is the way to make a test fail and is supposed to have a better report than a throw, isn't it?

Reply:
Oh my mistake, throwing seems the way to go now with Jest 👍 
https://github.com/facebook/jest/issues/11698